workflow/rules/common.smk

```diff
@@ -142,4 +151,5 @@ def compile_output_list(wildcards):
         "references/purecn_normal_db/output/normalDB.rds",
         "references/purecn_normal_db/output/mapping_bias.rds",
         "references/purecn_interval_file/targets_intervals.txt",
+        "references/jumble_reference/twist_DNA_solid.chr1.annotated.bed.reference.RDS",
```

Comment:
Very small thing. Should the suffix for these jumble reference and counts files be '.RDS' or '.rds'? Seems that the purecn files end in '.rds'?